core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -482,7 +490,10 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
 
         // Decrease the sponsor's collateral, tokens, and withdrawal request.
         _removeCollateral(positionData.rawCollateral, collateralToRemove);
-        positionData.tokensOutstanding = positionData.tokensOutstanding.sub(tokensToRemove);
+        FixedPoint.Unsigned memory newTokenCount = positionData.tokensOutstanding.sub(tokensToRemove);
+        require(newTokenCount.isGreaterThanOrEqual(minSponsorTokens));
```

Comment:
This function `_reduceSponsorPosition` is called by Liquidatable's `createLiquidation` function and so this indirectly places a restriction onto the minimum liquidation size based on how much collateral is left in the position after a liquidation. Does this introduce any strange behaviour with small partial liquidations?

Reply:
I think the strange behavior it induces is a weird discontinous liquidation function. For example: 

Let's say the sponsor has a 10 token position. The min position size is 5 tokens. You are allowed to liquidate 0-5 tokens OR 10 tokens, but not 5-9.99 tokens.

Reply:
this's fine, tbh. as long as it's not 99% and not below the threashold this defines.